apirest.md

```diff
@@ -605,6 +605,7 @@ Note: To download a document see [Download a document file](#download-a-document
   * *searchText* (default NULL): array of filters to pass on the query (with key = field and value the text to search)
   * *is_deleted* (default: false): Return deleted element. Optional.
   * *add_keys_names*: Retrieve friendly names. Array containing fkey(s) and/or "id". Optional.
+  * *with_networkports*: Retrieve all network's connections and advanced network's informations. Optional.
```

Comment:
```suggestion
  * *with_networkports*: Retrieve all network connections and advanced information. Optional.
```

Reply:
Ok, I've updated others places where this args was documented so we have the same everywhere.